LuaRules/Gadgets/game_share_mode.lua

```diff
@@ -313,9 +396,16 @@ function gadget:GameFrame(frame)
 		MergeAll()
 		firstMintime = false
 	end
+	if deadplayers.count > 0 then
+		for id,_ in pairs(deadplayers.players) do
+			DisposePlayer(id)
+			deadplayers.players[id] = nil
+		end
+		deadplayers.count = 0
+	end
```

Comment:
could be an array instead of using the IDs as indices (faster to iterate and you can avoid clearing)
```suggestion
	if deadplayersCount > 0 then
		for i = 1, deadplayersCount do
			DisposePlayer(deadplayers[i])
			-- not setting to nil, won't shrink the array anyway
		end
		deadplayersCount = 0
	end
```

Reply:
Creative.